types/react-native-keychain/react-native-keychain-tests.ts

```diff
@@ -0,0 +1,15 @@
+import * as Keychain from 'react-native-keychain';
+
+async () => {
+  const username = 'username';
+  const password = 'password';
+
+  await Keychain.setGenericPassword(username, password);
```

Comment:
`await` on its own line doesn't really test the types, since anything can be awaited -- would add `// $ExpectType boolean` after this. same for the other `await`s.